package.json

```diff
@@ -17,9 +17,13 @@
   },
   "homepage": "https://github.com/ukparliament/rss-notification-service#readme",
   "dependencies": {
+    "aws-sdk": "^2.342.0",
     "axios": "^0.18.0",
+    "chai": "^4.2.0",
     "cheerio": "^1.0.0-rc.2",
+    "local-dynamo": "^0.5.0",
```

Comment:
Are all of these dependencies production, or can some like this be moved to dev dependencies?